docs/tables/crowdstrike_spotlight_vulnerability.md

```diff
@@ -0,0 +1,95 @@
+# Table: crowdstrike_spotlight_vulnerability
+
+Vulnerabilities are known security risks in an operating system, application, hardware, firmware, or other part of a computing stack. Spotlight tracks vulnerabilities by industry standard frameworks like Common Vulnerabilities and Exposures (CVE). Spotlight identifies and gives info on specific vulnerabilities on your hosts using the Falcon sensor. Spotlight APIs help you integrate Spotlight into your organization's other ticketing or risk management tools.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  created_timestamp,
+  host_info -> 'hostname' AS hostname,
+  status
+from
+  crowdstrike_spotlight_vulnerability
+```
+
+### List all known vulnerabilities
+
+```sql
+select
+  created_timestamp,
+  host_info -> 'hostname' AS hostname,
+  status,
+  app -> 'product_name_version' AS product_name_version,
+  cve -> 'exprt_rating' AS exprt_rating,
+  cve -> 'id' AS cve_id,
+  cve -> 'description' AS cve_description
+from
+  crowdstrike_spotlight_vulnerability
+```
+
+### List vulnerabilities created in the last 15 days
+
+```sql
+select
+  created_timestamp,
+  host_info -> 'hostname' AS hostname,
+  status
+from
+  crowdstrike_spotlight_vulnerability
+where
+  created_timestamp > (now() - interval '15days')
```

Comment:
```suggestion
  created_timestamp > (now() - interval '15 days')
```

Do we use the formatting in `15 days` or `15days` more frequently across other examples?